src/cli.rs

```diff
@@ -74,6 +74,10 @@ pub(crate) struct Cli {
     #[arg(short = 'c', long = "clear-cache")]
     pub clear_cache: bool,
 
+    /// Override config file location
+    #[arg(long = "config", value_name = "FILE")]
+    pub config: Option<PathBuf>,
```

Comment:
In the issue, I was worried that we might want to have `--config` for something like `tldr --config 'cache.auto_update = true'`, that is, have `--config <some setting>` at some point. For example, git has a `-c` flag which works this way. Thus, `--config-path` would make more sense for the flag we are implementing here. However, I am not sure whether we will ever have such a flag ourselves, and `--config` is more convenient in the meantime. Still, I suspect that `--config-path` is not _much_ longer and is clearer to understand either way, so I am still learning towards that.

What do you think?

cc @dbrgn